src/components/projects/project-edit-form/project-edit-container/project-edit-container.tsx

```diff
@@ -39,10 +57,12 @@ const ProjectEditContainer: React.FC<EditFormContainerProps> = ({ wbsNum, proj,
   const [rules, setRules] = useState(proj!.rules);
   const { isLoading, isError, data, error } = useAllUsers();
 
+  const notEmptyString = (val: string) => val !== "";
```

Comment:
with type inference, you might be able to drop the type annotation but not sure, just wondering
```suggestion
  const notEmptyString = s => s !== '';
```

Reply:
also do you have prettier styling applied?